lib/src/signed_video_internal.h

```diff
@@ -144,11 +144,9 @@ struct _signed_video_t {
   bool is_golden_sei;  // Flag that tells if a SEI is a golden SEI
   bool using_golden_sei;  // Flag that tells if golden SEI prinsiple is used
   bool signing_started;
-  bool generate_sei;  // A flag that tells whether to generate SEI
-  // SEI is not generated at the start of the stream, but after the first GOP ends, this flag is set
-  // to true From that point, SEI is generated at the end of each GOP and includes the GOP hash The
-  // flag remains true until the stream ends, allowing SEI to be generated as long as the GOP hash
-  // is available.
+  bool first_signing_attempt;  // Flag indicating whether to generate the first SEI. Flips to true
```

Comment:
But if it flips to true after the first signing attempt the name does not match what it does. Either find a better name or flip from true to false. See further comments where it is used.